packages/framework/src/Form/Constraints/NotNegativeMoneyAmount.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class NotNegativeMoneyAmount extends Constraint
+{
+    /**
+     * @var string
+     */
+    public $message = 'Amount of money should be greater than or equal to zero.';
```

Comment:
ou... I expect protected... why public?

Reply:
Constraints have their properties public. I'd say it has something to do with how they are used in Symfony forms internally...